analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -202,217 +206,273 @@ def add_arguments_to_parser(parser):
+            checker_info[ClangTidy.ANALYZER_NAME].append(
+                (CheckerState.default, warning, ClangTidy.ANALYZER_NAME,
+                 'MEDIUM', guideline, ''))
 
-    if args.guideline is not None and len(args.guideline) == 0:
-        result = defaultdict(set)
+    return checker_info
 
-        for _, guidelines in context.guideline_map.items():
-            for guideline, rules in guidelines.items():
-                result[guideline] |= set(rules)
 
-        header = ['Guideline', 'Rules']
-        if args.output_format in ['csv', 'json']:
-            header = list(map(uglify, header))
+def __print_profiles(args, cl):
+    if 'details' in args:
+        header = ['Profile name', 'Description']
+        rows = cl.get_description('profile').items()
+    else:
+        header = ['Profile name']
+        rows = [(key,) for key in cl.get_description('profile')]
+
+    if args.output_format in ['csv', 'json']:
+        header = list(map(__uglify, header))
+
+    print(twodim.to_str(args.output_format, header, rows))
+
+
+def __print_guidelines(args, cl):
+    result = {}
+
+    for guideline in cl.get_description('guideline'):
+        result[guideline] = set(cl.occurring_values(guideline))
+
+    header = ['Guideline', 'Rules']
+    if args.output_format in ['csv', 'json']:
+        header = list(map(__uglify, header))
+
+    if args.output_format == 'json':
+        rows = [(g, sorted(list(r))) for g, r in result.items()]
+    else:
+        rows = [(g, ', '.join(sorted(r))) for g, r in result.items()]
+
+    if args.output_format == 'rows':
+        for row in rows:
+            print('Guideline: {}'.format(row[0]))
+            print('Rules: {}'.format(row[1]))
+    else:
+        print(twodim.to_str(args.output_format, header, rows))
+
+
+def __format_guideline(guideline):
+    """
+    Convert guideline rules to human-readable format.
+    guideline -- Dictionary in the following format:
+                    {"guideline_1": ["rule_1", "rule_2"]}
+    """
+    return ' '.join('Related {} rules: {}'.format(g, ', '.join(r))
+                    for g, r in guideline.items())
 
-        if args.output_format == 'json':
-            rows = [(g, sorted(list(r))) for g, r in result.items()]
-        else:
-            rows = [(g, ', '.join(sorted(r))) for g, r in result.items()]
 
-        if args.output_format == 'rows':
-            for row in rows:
-                print('Guideline: {}'.format(row[0]))
-                print('Rules: {}'.format(row[1]))
+def __format_row(row, format):
+    if format == 'json':
+        state = row[0] == CheckerState.enabled
+        guideline = row[4]
+    else:
+        state = '+' if row[0] == CheckerState.enabled else \
+            '-' if row[0] == CheckerState.disabled else \
+            '?'
+        guideline = __format_guideline(row[4])
+
+    return (state, row[1], row[2], row[3], guideline, row[5])
+
+
+def __print_checkers(args, cl):
```

Comment:
I tried to run the following command and I got an exception:
```
$ CodeChecker checkers --label profile
Traceback (most recent call last):
  ...
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_common/checker_labels.py", line 75, in __get_label_key_value
    pos = key_value.index(':')
ValueError: substring not found
```

This should be handle nicely and print an error message if I am using wrongly.